CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+## v1.2.1
+
+[Documentation](https://github.com/artsy/hokusai/blob/v1.3.0/README.md)
```

Comment:
v1.2.1?

Reply:
I didn't catch this on the `prepare-` PR. Let's leave it for now and fix it in the next release.